lib/global-admin/addon/accounts/edit/controller.js

```diff
@@ -43,7 +49,13 @@ export default Controller.extend(NewOrEdit, {
     set(this, 'errors', null);
 
     let errors  = [];
-    let ok      = this.validate();
+    let ok      = true;
+
+    ok = this.validate();
```

Comment:
I suspect you had this for testing. You should be able to restore this to the existing code since they're effectively equivalent in this context.

Reply:
Pretty close, changed it while refactoring and missed backing it out when the path I took didn't work out.